vendor/update-spawn.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-version=d5991f7eb4073c85b440789587936d14b4e0417f
+version=6c752122070f377c2607f6969a821f166a43bd5e
```

Comment:
Is this part of `janestreet/spawn`? It seems to be [a different fork](https://github.com/janestreet/spawn/commit/6c752122070f377c2607f6969a821f166a43bd5e) so this will fail on others who haven't pulled your branch. I suggest changing the URL to your fork temporarily for this issue until the fix is merged.

Reply:
We point at ocaml-dune/spawn. I've made this PR by running the update script so it will work for others too.
(in addition to this patch, the branch has a macos fix which is an unmerged upstream PR)

Reply:
Ah, fair point. I looked at the wrong place.